chef/cookbooks/ceph/recipes/osd.rb

```diff
@@ -71,7 +71,7 @@
   if is_crowbar?
     dirty = false
 
-    node.set["ceph"]["osd_devices"] ||= []
+    node.set["ceph"]["osd_devices"] = [] if node["ceph"]["osd_devices"].nil?
```

Comment:
Sorry, I am still not completely comfortable with the `||=` notation, but do I interpret your change correctly that it doesn't change anything in terms of what the code does? If that's the case, then why change it?